### ISSUE TITLE
Bootable win10 USB stick created with woeUSB is detected by BIOS but nothing happens when trying to boot from it

### ISSUE THREAD

**Issue by Dehde** [slacka/WoeUSB#195]
The general issue can be found here as well: https://askubuntu.com/questions/1042759/bootable-win10-usb-stick-created-under-ubuntu-results-in-black-screen?noredirect=1#comment1699305_1042759

The text, copied:

I want to get a dual boot setup with Ubuntu 18 and Windows 10. I am currently running Ubuntu 16.04. My plan was to first install win 10 then install ubuntu 18 over the old Ubuntu version.

I created a bootable USB stick following the instructions are given in the following question and answer: https://askubuntu.com/questions/289559/how-can-i-create-a-windows-bootable-usb-stick-using-ubuntu/487970#487970

I create the bootable USB stick using woeUSB, before installing that I created the gpt partition table and then formatted the stick to fat32 as demanded. I then restart the computer, go into BIOS, go to the boot override and boot from the USB. At first, I see the windows symbol for waiting (the dots that are moving in a circle), but the dots quickly disappear. Then nothing happens and I stare into a black screen.

I tried several ways that are proposed in the answer I mentioned above, yet the only results I achieve are either the one I just described, or the USB stick is not even recognized.

I also tried creating the bootable USB stick using mkusb, as suggested in my askubuntu question. This yields the same results as with using woeUSB.

I do not know what else to do. Thanks a lot in advance for your help!!

My setup: Running ubuntu 16.04 on an Asus gaming laptop (G501V). The stick I am trying to use is from SanDisk with 64 GB storage.

### WoeUSB Source
> Built from source code, commit abcdefg
> Installed from Ubuntu 16.04 software archive
> Installed from WebUpd8 PPA

### GNU Bash Version
> WoeUSB exploits several advanced features of the Bash scripting language and requires a relatively-recent Bash intepreter.  Run `bash --version` in a terminal to acquire this information.
The above results in: GNU bash, version 4.3.48(1)-release (x86_64-pc-linux-gnu)

### Information about the Operating System
> Different OS distributions provides different characteristics and may influence the result of running WoeUSB.  Run `lsb_release --description` or `lsb_release -d` in a terminal to acquire this information.
Results in: Ubuntu 16.04.4 LTS

### Information about the Source Media
Windows downloaded from: https://www.microsoft.com/en-us/software-download/windows10ISO

### Information about the Target Device
SanDisk Extreme (0001), 64 GB

Thank you for your time and help!!

**Comment by WaxyMocha** [comment]
And make sure that you have unmounted your flash drive before. Use umount command or disks program

**Comment by Lin-Buo-Ren** [comment]
This seems to be rather a compatibility issue of Windows instead of WoeUSB...  As a last resort please make the target device with Rufus on a Windows PC to see if the same problem can still be reproduced.

**Comment by Lin-Buo-Ren** [comment]
Also please try to boot the same target device on other PCs to verify that it can boot correctly or not.

> I create the bootable USB stick using woeUSB, before installing that I created the gpt partition table and then formatted the stick to fat32 as demanded.

WoeUSB doesn't support GUID Partition Table configuration at the moment.

**Comment by Dehde** [comment]
So, it turns out that my problem had to do with my own harddisk. I did have a dual boot before with Windows 10 and ubuntu 16.04. At some point though, weirdly, windows 10 just stopped working. Just couldn't boot it anymore. Since I rarely used it I just left it at that.

This faulty windows10 installation was my problem here though. Only after I formatted the part of the harddisk where Windows10 lay was it possible to get into the installation menu of windows10. At that point all usb sticks I created started working.

Sorry for bothering you guys and thanks for your help!! :)
